.github/workflows/stale.yml

```diff
@@ -0,0 +1,18 @@
+name: stale
+
+on:
+  schedule:
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
+
+jobs:
+  mark-stale:
+    uses: openclarity/.github/.github/workflows/_stale.yml@main
+    permissions:
+      issues: write
+      pull-requests: write
+    with:
+      days-until-stale: 60
```

Comment:
This looks great! The 60 days default is quite realistic though I think a 90-day period seems a more generous and reasonable approach to take a quarterly perspective into the stale issues. I'm fine with the current default as long as this number can be altered per project community engagement patterns that emerge over time.

Reply:
Appreciate the input! We could certainly adjust to 90 days in the future if 60 still feels too short. With the current configuration, issues will still remain open for 75 days _total_, which feels like a reasonable starting point. 

But yes, these values will be trivial to change in the future once the new pattern is in place 🙌